c/xcsp/AllInterval-014.c

```diff
@@ -0,0 +1,340 @@
+extern void abort(void) __attribute__((__nothrow__, __leaf__))
+__attribute__((__noreturn__));
+int __VERIFIER_nondet_int();
+void reach_error() {}
+void assume(int cond) {
+  if (!cond)
+    abort();
+}
+int dist(int a, int b) {
```

Comment:
unused method?

Reply:
I adapted the python script to remove the method dist from every file containing the string "dist" only one time. Additionally, I replaced os.system with subprocess.